resources/chef_ingredient.rb

```diff
@@ -103,8 +103,15 @@
       end
     end
 
+    reconfigure_command = case new_resource.accept_license && new_resource.product_name == 'chef-server'
```

Comment:
This seems better as a method in the action_class

```ruby
def reconfigure_command
  if new_resource.accept_license && new_resource.product_name == 'chef-server'
    'reconfigure --chef-license=accept'
  else
    'reconfigure'
  end
end
```

Reply:
Thanks for the suggestion. Just updated my branch.